xarray/tests/test_dataarray.py

```diff
@@ -2940,6 +2942,39 @@ def test_to_and_from_cdms2(self):
         roundtripped = DataArray.from_cdms2(actual)
         assert_identical(original, roundtripped)
 
```

Comment:
Could you add separate test methods for this? It makes things a little easier to debug when things go wrong :).